### PR TITLE
Cody vscode v1.10.0 blog post

### PR DESCRIPTION
**Pull request by kalanchan**
VS Code v1.10 blog post

### CODE REVIEW

File: content/blogposts/2024/cody-vscode-1-10-0-release.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  alt="Claude3 Haiku in the LLM Selection dropdown"
+/>
+
+## Better Debugging for Cody
+
+We've added new commands and shortcuts to make debugging Cody easier, these improvements enable debugging and accessing logs. 
+
+- Debug commands now grouped in command palette under "Cody Debug"
+- New sidebar menu item to open the output channel
+- New settings menu button to enable debug mode and output channel
```
Comment (toolmantim) [inline content/blogposts/2024/cody-vscode-1-10-0-release.md:56]:
```suggestion
- Debug commands are now grouped in command palette under "Cody Debug"
- Added an "Open Output Channel" sidebar menu item, that opens the VS Code output panel with "Cody by Sourcegraph" selected
- Added a debug button in the top right of the Cody settings panel, which sets `{ "cody.debug.verbose": true }` and opens the VS Code output panel
```

^ @abeatrix is that correct

^ (also why do we have two ways of doing the same thing, and that they're slightly different?)

Comment (abeatrix) [reply content/blogposts/2024/cody-vscode-1-10-0-release.md:56]: The sidebar is not available to users who can't / haven't logged in, is that what you're referring to (sidebar vs menu)?

File: content/blogposts/2024/cody-vscode-1-10-0-release.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+## Improved Document Command
+
+Generating documentation from code has been improved to automatically detect documentable code blocks and provide hints to quickly generate doc strings. When your cursor is on a documentable symbol in JavaScript, TypeScript, Go or Python, a "Opt/Alt+D to Document" ghost text will appear. Pressing Opt/Alt+D will immediately generate documentation for that code block.
+
+<Figure
+  src="https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1.10.0/document-command.png"
+  alt="Document Command"
+/>
+
+## Claude 3 Haiku is now available for Cody Pro
```
Comment (philipp-spiess) [inline content/blogposts/2024/cody-vscode-1-10-0-release.md:41]: If we mention Haiku should we also mention that we've updated GPT 4 Turbo to the latest version?

Comment (kalanchan) [reply content/blogposts/2024/cody-vscode-1-10-0-release.md:46]: good point!

File: content/blogposts/2024/cody-vscode-1-10-0-release.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+## Claude 3 Haiku is now available for Cody Pro 
+
+For Cody Pro users we've added Anthropic’s latest AI model, [Claude 3 Haiku](https://www.anthropic.com/news/claude-3-haiku), alongside Sonnet and Opus. This new model offers significantly faster speeds while maintaining quality and coherence. To start using Claude 3 Haiku start a new chat and select it from the list of models:
+
+<Figure
+  src="https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1.10.0/claude3-haiku.png"
+  alt="Claude3 Haiku in the LLM Selection dropdown"
+/>
+
+## New debug options
```
Comment (abeatrix) [inline content/blogposts/2024/cody-vscode-1-10-0-release.md:58]: Personally think this should be grouped with the `Support links for Cody Pro and Enterprise` entry above, and mention attaching the logs when they file a support ticket can speed things up etc, and then proceed with how they can get the logs with these new Debug command. wdyt?

**Pull request merged**